app/uk/gov/hmrc/cdsreimbursementclaimstubs/controllers/SubmitClaimController.scala

```diff
@@ -36,7 +36,7 @@ class SubmitClaimController @Inject() (cc: ControllerComponents) extends Backend
       Json.parse(
         Source
           .fromInputStream(
-            this.getClass.getResourceAsStream("/resources/submit-claim-schema-1-0.json")
+            this.getClass.getResourceAsStream("/resources/TPI05-v1-0-0-submit-claim-schema.json")
```

Comment:
Just need to change the schema loader reference in the submit controller file

Reply:
What? 
It is changed!
The filename and the reference were both changed